connectors/ssh/src/main/java/org/crsh/ssh/SSHPlugin.java

```diff
@@ -82,13 +85,18 @@ public SSHPlugin getImplementation() {
 
   @Override
   protected Iterable<PropertyDescriptor<?>> createConfigurationCapabilities() {
-    return Arrays.<PropertyDescriptor<?>>asList(SSH_PORT, SSH_SERVER_KEYPATH, SSH_SERVER_KEYGEN, SSH_SERVER_AUTH_TIMEOUT,
+    return Arrays.<PropertyDescriptor<?>>asList(SSH_HOST, SSH_PORT, SSH_SERVER_KEYPATH, SSH_SERVER_KEYGEN, SSH_SERVER_AUTH_TIMEOUT,
         SSH_SERVER_IDLE_TIMEOUT, SSH_ENCODING, AuthenticationPlugin.AUTH);
   }
 
   @Override
   public void init() {
 
+    String host = getContext().getProperty(SSH_HOST);
+    if (host == null) {
+      log.log(Level.INFO, "Missing host configuration, using default host");
```

Comment:
The host configuration was never provided before though. So I don't think we can issue this message now.
What about something like host configuration not provided, and have the debug level set to DEBUG.